lib/node_modules/@stdlib/math/base/napi/unary/include/stdlib/math/base/napi/unary/b_b.h

```diff
@@ -0,0 +1,85 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_MATH_BASE_NAPI_UNARY_B_B_H
+#define STDLIB_MATH_BASE_NAPI_UNARY_B_B_H
+
+#include <node_api.h>
+#include <assert.h>
+#include <stdint.h>
+
+/**
+* Macro for registering a Node-API module exporting an interface invoking a unary function accepting and returning 8-bit unsigned integers.
+*
+* @param fcn   unary function
+*
+* @example
+* #include <stdint.h>
+*
+* static uint8_t scale( const uint8_t x ) {
+*     return x * 10;
+* }
+*
+* // ...
+*
+* // Register a Node-API module:
+* STDLIB_MATH_BASE_NAPI_MODULE_B_B( scale );
+*/
+#define STDLIB_MATH_BASE_NAPI_MODULE_B_B( fcn )                                \
```

Comment:
@gunjjoshi All of the `\` should be column-aligned. Ideally, this is at the 80-char column limit.